caluma/caluma_form/jexl.py

```diff
@@ -61,6 +62,22 @@ def answer_transform(self, question_slug):
         if self.is_hidden(field):
             return field.question.empty_value()
 
+        # This overrides the logic in field.value() to consider visibility for
+        # table cells
+        elif (
+            field.question.type == Question.TYPE_TABLE
+            and field.answer is not None
+            and field.answer.value is None
```

Comment:
Not sure we should even do that last check (`answer.value is None`) If some code didn't keep proper data hygiene, this may return wrong data

Reply:
True, I did this check because it's what would happen in `field.value()` so I wanted to be consistent. I'll remove it

Reply:
done